lib/pbench/server/api/resources/api_key.py

```diff
@@ -63,15 +117,52 @@ def _post(
             raise APIInternalError(str(e)) from e
 
         try:
-            key = APIKey(api_key=new_key, user=user)
+            key = APIKey(api_key=new_key, user=user, name=name)
```

Comment:
Good.

However, your comment brings up an important nuance:  I think that this field **is** in fact a _description_ of the key, and not a "name" for it (whatever that might mean).  So, perhaps it would help if we renamed the field to "description".

Reply:
Yeah, I agree that the `name` is not the correct tag. But instead of a `description`, there are a few other suggestions that some of the other product uses (i.e.) `label` or `title`. Or else we can finalize with the `description` itself? @webbnh any inputs on this?

Reply:
Personally, I like `description`, but I think `label` would be fine.  (I don't much like `title` for this, but it's better than `name`, I think.  🙂)